src/model.ts

```diff
@@ -994,7 +1011,6 @@ export class HeatmapcustomColumn extends ValueColumn<number[]> {
       return result;
     }
 
-
     const a1_sum = d3.sum(a_val);
```

Comment:
computing all different values for all different possible sort criteria is inefficient